charts/backingservices/README.md

```diff
@@ -1,30 +1,27 @@
 # BackingServices Helm chart
 
-The Pega Infinity backing service is a feature which you can deploy as an independent service module. For example `Search and Reporting Service` or `SRS` backing service can replace the embedded search feature of Pega Infinity Platform. To use it in your deployment, you provision and deploy it independently as an external service which provides search and reporting capabilities with one or more Pega Infinity environments.  
+The Pega Infinity backing service is a feature which you can deploy as an independent service module. For example `Search and Reporting Service` or `SRS` backing service can replace the embedded search feature of Pega Infinity Platform. To use it in your deployment, you provision and deploy it independently as an external service which provides search and reporting capabilities with a Pega Infinity environment.  
 
-The backingservices chart supports deployment options for `Search and Reporting Service` (abbreviated as SRS). A backing service may be configured with one or more Pega deployments. 
-These backing services should be deployed in their own namespace and may be shared across multiple Pega Infinity environments.
+The backingservices chart supports deployment option for `Search and Reporting Service` (abbreviated as SRS). A backing service may be configured with a Pega deployments. 
+These backing services may be deployed in to the pega namespace and configured with the Pega Infinity Environment.
```

Comment:
The backingservices chart supports deployment option**s** for `Search and Reporting Service` (SRS). You configure the SRS into the pega namespace for your Pega Infinity deployment.